device_calendar/android/src/main/kotlin/com/builttoroam/devicecalendar/CalendarDelegate.kt

```diff
@@ -348,12 +341,12 @@ public class CalendarDelegate : PluginRegistry.RequestPermissionsResultListener
         if (arePermissionsGranted()) {
             var existingCal = retrieveCalendar(calendarId, pendingChannelResult, true)
             if (existingCal == null) {
-                finishWithError(CALENDAR_RETRIEVAL_FAILURE, "Couldn't retrieve the Calendar with ID ${calendarId}", pendingChannelResult)
+                finishWithError(NOT_FOUND, "Couldn't retrieve the Calendar with ID ${calendarId}", pendingChannelResult)
                 return
             }
 
             if (existingCal.isReadyOnly) {
-                finishWithError(CALENDAR_IS_READ_ONLY, "Calendar with ID ${calendarId} is read only", pendingChannelResult)
+                finishWithError(NOT_ALLOWED, "Calendar with ID ${calendarId} is read only", pendingChannelResult)
```

Comment:
405 represents a method that doesn't exist. I reckon it should be 403. i'll take a look at the iOS code and see will add that in

Reply:
Where do you get the info about the statuses? Wikipedia says "405 Method Not Allowed
A request method is not supported for the requested resource". 

403 is more related to missing permissions - I would say

https://en.wikipedia.org/wiki/List_of_HTTP_status_codes

Reply:
hmm i guess could argue both ways as read-only means it could just be only by the OS itself or maybe it's another person's calendar (assuming it's possible to get other users calendars, i don't know if that can be done). guess leave it as it is then